project-2-e-commerce/src/App.js

```diff
@@ -0,0 +1,28 @@
+import Header from './component/Header'
+import Footer from './component/Footer'
+import Home from './component/Home'
+import Products from './component/Products'
+import ProductsDetail from './component/ProductsDetail'
+import Login from './component/Login'
+import Register from './component/Register'
+import { Routes, Route } from "react-router-dom";
+
+function App() {
+  return (
+    <>
+        <Header></Header>
+        <div className='main'>
+            <Routes>
+                <Route path="/" element={<Home />}/>
+                <Route path="products" element={<Products />}/>
+                <Route path="products/:productid" element={<ProductsDetail />}/>
+                <Route path="login" element={<Login />}/>
+                <Route path="register" element={<Register />}/>
```

Comment:
Thêm 1 Route cho page 404 not found nữa anh nhé

Reply:
Oke thank you em nha!